factorio-docker-watchdog.go

```diff
@@ -136,24 +164,19 @@ func updateVersion(version semver.Version) {
 	//}
 	//logrus.Info("Checkout branch ", version)
 
-	checksum, err := factorioGetChecksum(fmt.Sprintf("https://www.factorio.com/get-download/%s/headless/linux64", version))
-	if err != nil {
-		logrus.Panic(err)
-	}
-	logrus.Info("Got checksum ", checksum)
+	//err = editReadme(pathRepo, buildinfo)
+	//if err != nil {
+	//	logrus.Panic(err)
+	//}
+	//logrus.Info("Edited README")
 
-	err = editDockerfile(pathRepo, version, checksum)
-	if err != nil {
-		logrus.Panic(err)
-	}
-	logrus.Info("Edited Dockerfile")
-	err = editReadme(pathRepo, version)
+	err = editBuildinfo(pathRepo, buildinfo)
 	if err != nil {
 		logrus.Panic(err)
 	}
 	logrus.Info("Edited README")
 
-	err = gitCreateCommit(pathRepo, "Update to Factorio "+version.String())
+	err = gitCreateCommit(pathRepo, "Update to Factorio version")
```

Comment:
This is currently missing the version that has been committed.
Logic for this could be to compare the old `buildinfo.json` for the highest version, and if that did not change compare the versions with `stable` tags.

Reply:
Yea i was thinking about it, but because of the changes that could happen:
- stable tag moves
- new version is released
- ....

I choosed to use a basic commit message